src/Abr.cpp

```diff
@@ -61,29 +61,32 @@ struct Abr : Module
 
     bool swState[8] = {};
     
-	Abr() : Module(NUM_PARAMS, NUM_INPUTS, NUM_OUTPUTS, NUM_LIGHTS) 
-	{
-		reset();
-	}
+    Abr() {
+
+      config(NUM_PARAMS, NUM_INPUTS, NUM_OUTPUTS, NUM_LIGHTS);
+      reset();
+
+      configParam(Abr::SWITCH1_PARAM, 0.0, 1.0, 0.0, "");
+    }
 
-    void step() override;
+    void process(const ProcessArgs &args) override;
 
-    void reset() override 
+    void reset()
```

Comment:
Several modules overrode `reset()` however it's undetermined if this needs to be changed to `onReset` or not. Same goes for `randomize()` methods.

What stopped me from modifying these methods more intentionally is the fact that some modules depend on being able to call `reset()` in their constructors.